addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9646,7 +9646,7 @@ msgstr ""
 #. DialogOK for no installed ADSP add-on available
 #: xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp
 msgctxt "#19274"
-msgid "You need a add-on installed for the process of audio DSP signal. System becomes disabled."
+msgid "You need a add-on installed for the process of audio DSP signal. Audio DSP is disabled again."
```

Comment:
An alternative wording:

"For Audio DSP to be enabled, there must be at least one Audio DSP add-on installed, since there are no Audio DSP add-on's installed, Audio DSP has been disabled again."